Frontend/android-native/mobility-customer/src/main/user/mobility_rs/assets/juspay/v1-configuration.js

```diff
@@ -49,8 +76,13 @@ window.getMerchantConfig = function () {
       "searchLocationConfig": {
         "searchLocationTheme": "#D84E55",
         "setLocationOnMapColor": "#2C2F3A",
-        "strokeColor": "1,#13101010",
-        "enableLocationTagbar": "true",
+        "strokeColor": "1,#818181",
+        "enableLocationTagbar" : "true",
+         "editTextBackground" : "#EFEFEF",
```

Comment:
values are repeated. Update the config properly. Ex. footer visibility is not there